More Random Edition/AssetEditor.cs

```diff
@@ -160,30 +162,35 @@ public void Edit<T>(IAssetData asset)
 			{
 				this.ApplyEdits(asset, this._preferenceReplacements);
 			}
+			else if (asset.AssetNameEquals("Data/SecretNotes"))
+            {
```

Comment:
Looks like there's a few tabs vs. spaces issues. Not a huge deal, though :P

Reply:
Thanks, I never see these on my end 🙄 think I found the setting in Visual Studio that was causing it though!